### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by houseburger**
google maps & filter for current month

### CODE REVIEW

File: site/templates/calendar.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?php snippet('header') ?>
+<?php snippet('menu', array('color' => 'dark-gray')) ?>
+
+<div class="py4 container clearfix ">
+  <header class=" center col-8 mx-auto mb4">
+    <p class="h1 block letter-spacing pb2"><?= $page->title() ?></p>
+    <p class="h3 lighter mx-auto col-inline mb3"><?= $page->text() ?></p>
+  </header>
+  <div class="questions grid">
```
Comment (timothyachumba) [inline site/templates/calendar.php:9]: This div needs to be renamed

**Comment by timothyachumba** [comment]
@hauspa Hey man, great work. It seems there are some conflicts though. It's probably really small. Can you take a look? Once you're done I will merge and deploy a version.

**Comment by timothyachumba** [comment]
The conflicts resolved themselves

**Comment by houseburger** [comment]
oh man, i'm sorry i didn't see it in time, tim! :/
thanks for resolving it!
see you next week!
pasci

**Pull request merged**